.travis.yml

```diff
@@ -0,0 +1,98 @@
+sudo: false
+
+language: cpp
+
+addons:
+  apt:
+    sources: &toolchains
+      - ubuntu-toolchain-r-test
+      - llvm-toolchain-trusty-5.0
+
+matrix:
+  include:
+    - os: linux
+      addons:
+        apt:
+          packages:
+            - g++-7
+          sources:
+            - *toolchains
+      compiler: g++-7
+      script: b2 -j3 libs/spirit/test toolset=gcc-7 cxxstd=14
+
+    - os: linux
+      addons:
+        apt:
+          packages:
+            - clang++-5.0
+            - libc++-dev
+          sources:
+            - *toolchains
+      compiler: clang++-5.0
+      script: b2 -j3 libs/spirit/test toolset=clang-5.0 cxxstd=14 cxxflags="-stdlib=libc++" linkflags="-stdlib=libc++"
+
+    - os: osx
+      compiler: clang++
+      script: b2 -j3 libs/spirit/test toolset=clang cxxstd=14
+
+install:
+  - cd ..
+  - git clone -b develop --depth 1 https://github.com/boostorg/boost.git boost-root
+  - cd boost-root
+  - export BOOST_ROOT="`pwd`"
+  - export PATH="`pwd`":$PATH
+
+  - git submodule update --init tools/build
```

Comment:
It would be easier to use the boostdep tool here so that future changes don;t cause this list to need to be updated manually.

See the install phase of the [Boost.Uuid .travis.yml ](https://github.com/boostorg/uuid/blob/develop/.travis.yml)for an example.

Reply:
nice hint, thanks! I will have a look...
on the other side: now we see the HUGE dependencies to other libs - this might be nice to know.